spec/integration/admin_items_spec.rb

```diff
@@ -6,7 +6,6 @@
 
     it "an admin can see an item listing page" do
       mock_admin
-
```

Comment:
Maybe you want to leave the blank line in there? To keep the separation between setup (mock_admin) and exercise (visit items_path).